external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -41,6 +41,7 @@ class DB2IntegrationSuite extends DockerJDBCIntegrationSuite {
     override val jdbcPort: Int = 50000
     override def getJdbcUrl(ip: String, port: Int): String =
       s"jdbc:db2://$ip:$port/foo:user=db2inst1;password=rootpass;retrieveMessagesFromServerOnGetMessage=true;" //scalastyle:ignore
+    override def getEntryPoint: Option[String] = None
```

Comment:
Why not have this in the base class and only override where needed?

Reply:
Good simplification, was thinking about this but not yet done. Fixed.

Reply:
FYI, I've applied the same on `getStartupProcessName`.